src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,8 +1,170 @@
+
+  const ref1 = useRef(null);
+  const ref2 = useRef(null);
+  const modal = useRef(null);
+
+  const scrollToElement = () => {
+    ref1.current.scrollIntoView();
+  };
+
+  const handleButton = () => {
+    ref2.current.scrollIntoView();
+  };
+  // const params = useParams();
+  useEffect(() => {
+    fetch('http://10.58.1.160:8000/products/16')
+      .then(res => res.json())
+      .then(data => setProduct(data.result));
+  }, []);
+
+  const navigate = useNavigate();
+  const goToProductList = () => {
+    navigate('/products');
+  };
+  const goToCart = () => {
+    navigate('/cart');
+  };
+
+  return (
+    <div className="product-detail">
+      <div className="product-detail-wrap" ref={ref1}>
+        <div className="product-detail-wrap-left">
+          <div className="product-detail-wrap-left__photo">
+            <img src={product.product_image} alt="product" />
+
+            <button type="button" onClick={goToProductList}>
+              <img src="/images/ProductDetail/left-arrow.png" alt="arrow" />
+              <span>스피커</span>
+            </button>
+          </div>
+        </div>
+
+        <Modal
+          modal={modal}
+          open={modalOpen}
+          close={closeModal}
+          header="상품이 장바구니에 추가됨"
+          product={product}
+          goToCart={goToCart}
+        >
+          <div className="modal-wrap">
+            <div className="modal-wrap-top">
+              <img src={product.product_image} alt="product" />
+              <div className="modal-wrap-top-name">
+                <p>{product.name}</p>
+                <p>Natural Oak</p>
+              </div>
+              <div className="modal-wrap-top-price">
+                ₩
+                {parseInt(product.price)
+                  .toString()
+                  .replace(/\B(?=(\d{3})+(?!\d))/g, ',')}
+              </div>
+            </div>
+            <div className="modal-wrap-bottom">
+              <p>관련 제품</p>
+              <div className="modal-wrap-bottom-detail">
+                <img
+                  src="/images/ProductDetail/portablespeaker.jpg"
+                  alt="product"
+                />
+                <div className="modal-wrap-bottom-detail-text">
+                  <p>Beosound Level</p>
+                  <p>
+                    언제나 나와 함께하는 레벨. 언제 어디서나 성능, 휴대성,
+                    연결성을 제공합니다. 세워서, 또는...
+                  </p>
+                  <a href="#!">자세히 보기</a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </Modal>
+
+        <div className="product-detail-wrap-right">
+          <div className="product-detail-wrap-right__name">{product.name}</div>
+          <div className="product-detail-wrap-right__short-desc">
+            <h2>{product.description}</h2>
+          </div>
+          <div className="product-detail-wrap-right__long-desc">
+            {`언제나 나와 함께하는 레벨. 언제 어디서나 성능, 휴대성, 연결성을 제공합니다. 
+            세워서, 또는 눕혀서, 내가 좋아하는 노래를 항상 내곁에.`}
```

Comment:
중간에 자바스크립트 변수가 사용되는 것이 아니라면 중괄호와 백틱 사용하지 않으셔도 될 것 같네요!
```suggestion
            언제나 나와 함께하는 레벨. 언제 어디서나 성능, 휴대성, 연결성을 제공합니다. 
            세워서, 또는 눕혀서, 내가 좋아하는 노래를 항상 내곁에.
```